pkg/addons2/openebs/install.go

```diff
@@ -0,0 +1,5 @@
+package openebs
+
+func (o *OpenEBS) Install() error {
```

Comment:
we might want to always just use `helm upgrade -i` if we ever want the installation process to be idempotent.

Reply:
not implemented here, but sure